Sources/CoreCommands/SwiftTool.swift

```diff
@@ -162,7 +162,7 @@ public final class SwiftTool {
     public let sharedConfigurationDirectory: AbsolutePath?
 
     /// Path to the cross-compilation SDK directory.
-    let sharedCCSDKDirectory: AbsolutePath?
+    let sharedDestinationsDirectory: AbsolutePath?
```

Comment:
nit: I would keep the cross compilation name here to make it easy to reason about

Reply:
Would "CC" as an abbreviation be ok here, as in `sharedCCDestinationsDirectory`? I remember @neonichu had an objection to "CC" as an abbreviation, as it can be confused with something like "C Compiler". OTOH `sharedCrossCompilationDestinationsDirectory` seems too verbose

Reply:
I am fan of explicit variable names (even if verbose) since they make maintenance / reading code much easier